js/index.js

```diff
@@ -1,9 +1,353 @@
-    const minutes = now.getMinutes().toString().padStart(2, '0');
-    const seconds = now.getSeconds().toString().padStart(2, '0');
-    return `${hours}:${minutes}:${seconds}`;
-}
-
-console.log(getCurrentTime());
\ No newline at end of file
+'use strict'
+{
+
+  const sysmember = [
+    { "src": "./images/otakesan.webp", "nickname": "おたけ", "value": "otake" },
+    { "src": "./images/riri-san.webp", "nickname": "リリー", "value": "riri-" },
+    { "src": "./images/mossan.webp", "nickname": "もっさん", "value": "mossan" },
+    { "src": "./images/shimopi-san.webp", "nickname": "しもぴー", "value": "shimopi-" },
+    { "src": "./images/oga00385.jpg", "nickname": "ななみん", "value": "nanamin" },
+    { "src": "./images/makottyan.webp", "nickname": "まこっちゃん", "value": "makottyan" },
+    { "src": "./images/utti-san.webp", "nickname": "うっちー", "value": "utti-" },
+    { "src": "./images/akuisan.webp", "nickname": "だいちゃん", "value": "daityan" },
+    { "src": "./images/yoshiokasan.webp", "nickname": "吉岡さん", "value": "yoshiokasan" },
+    { "src": "./images/nagayansan.webp", "nickname": "ながやん", "value": "nagayan" },
+    { "src": "./images/oga00331.webp", "nickname": "みっぴー", "value": "mippi-" },
+  ];
+  //ニックネームを回答するモード
+  let nicknameMode = true;
+  //顔を回答するモード
+  let faceMode = false;
+  //表示インデックス
+  let randomIndex;
+  //問題数を格納 最終問題かの判定を行う
+  const questionLength = sysmember.length;
+  //何問表示したかをカウント
+  let count = 0;
+  //結果格納 
+  let result;
+  //正解数カウント
+  let correctCount = 0;
+  //正答格納用
+  let correctAnswer;
+  //正解率
+  let accuracyRate; 
+  //アニメーション
+  let animation;
+
+  const gameControl = document.getElementById('gameControl'); // 開始ボタン
+  const image = document.getElementById('icon'); // ロゴ画像
+  
+  // スタートボタンをクリックしたらアイコンが動き出す
+  gameControl.addEventListener('click', () => {
+    if (gameControl.textContent === 'START') {
+      startAnimation();
+      //１回目の抽選
+      setQuestion();
+      //ボタン表示変更
+      gameControl.textContent = 'OK';
+      //成績欄を非表示
+      document.getElementById('performance').style.display = "none";
+      return;
+    };
+    if (gameControl.textContent === 'OK') {
+      let answer = checkAnswer();
+      if(answer === undefined){
+        alert("選択してください");
+        return;
+      };
+      result = judge(answer);
+      displaydialog(result);
+    };
+  });
+
+  //アニメーションを開始する関数
+  function startAnimation() {
+    animation = image.animate(
+      // 途中の状態を表す配列
+      [
+        { transform: 'translateX(0)' }, // 開始時の状態（左端）
+        { transform: 'translateX(1500px)' } // 終了時の状態（左端から200pxの位置）
+      ],
+      // タイミングに関する設定
+      {
+        fill: 'forwards', // 再生後、最後の状態を保持
+        duration: 60000, // 再生時間（1000ミリ秒）
+      },
+    );
+  }
+
+  //アニメーションを停止する関数
+  function stopAnimation() {
+    if (animation) {
+      animation.pause(); //一時停止
+    }
+  }
+
+  //表示する問題をランダムに抽選する関数
+  function getrandomIndex() {
+    randomIndex = Math.floor(Math.random() * sysmember.length);
```

Comment:
`sysmember.length`は上のほうで、一回変数に落しているからそれをつかえばいいかな。
```suggestion
    randomIndex = Math.floor(Math.random() * questionLength);
```